### PR TITLE
Azure doc only apps set2

### PR DESCRIPTION
**Pull request by himanshu219**
## Purpose of this pull request

This pull request (PR) contains 15 azure doc only apps

Issue number: 

[SUMO-226480](https://sumologic.atlassian.net/browse/SUMO-226480)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/microsoft-azure/azure-backup.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+<img src={useBaseUrl('img/integrations/microsoft-azure/azure-backup.png')} alt="Thumbnail icon" width="50"/>
+
+The [Azure Backup](https://learn.microsoft.com/en-us/azure/backup/backup-overview) service provides simple, secure, and cost-effective solutions to back up your data and recover it from the Microsoft Azure cloud. This integration helps you in getting rich insights on your backups across your entire backup estate.
+
+## Log and Metric types
+
+For Azure Backup, you can collect the following logs and metrics:
+
+* **Diagnostic logs**. Azure Backup provides the following diagnostics events:
+
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-backup.md:18]:
```suggestion
```

File: docs/integrations/microsoft-azure/azure-cache-for-redis.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+id: azure-cache-for-redis
+title: Azure Cache for Redis
+description: Learn about the Sumo Logic collection process for the Azure Cache for Redis service.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/azure-cache-for-redis.png')} alt="Thumbnail icon" width="50"/>
+
+[Azure Cache for Redis](https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-overview) provides an in-memory data store based on the Redis software. It offers both the Redis open-source (OSS Redis) and a commercial product from Redis Inc. (Redis Enterprise) as a managed service. This integration helps in tracking cache performance (miss rate, latency, read and write rate) and monitor resource usage (CPU, used memory, server load and connections) of your instances.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-cache-for-redis.md:11]:
```suggestion
[Azure Cache for Redis](https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-overview) provides an in-memory data store based on the Redis software. It offers both the Redis open-source (OSS Redis) and a commercial product from Redis Inc. (Redis Enterprise) as a managed service. This integration helps in tracking cache performance (miss rate, latency, read and write rate) and monitor resource usage (CPU, used memory, server load, and connections) of your instances.
```

File: docs/integrations/microsoft-azure/azure-cosmos-db-for-postgresql.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+[Azure Cosmos DB for PostgreSQL](https://learn.microsoft.com/en-us/azure/cosmos-db/postgresql/introduction) is a managed service for PostgreSQL powered by the Citus open source extension which enables you to build highly scalable relational apps. This integration helps in identifying configurations errors, analysing executed statements, and monitoring resource usage of individual nodes in a cluster.
+
+## Log and Metric types
+
+For Azure Cosmos DB for PostgreSQL, you can collect the following logs and metrics:
+
+* **PostgreSQL Server Logs**. These logs are available for every node of a cluster and can be used to identify, troubleshoot, and repair configuration errors and suboptimal performance.
+* **Audit Logs**. Audit logging of database activities is available through [pgAudit](https://www.pgaudit.org/) extension.By default, pgAudit log statements are emitted along with your regular log statements by using Postgres's standard logging facility. To know more about the audit log format, refer to the [pgAudit documentation](https://github.com/pgaudit/pgaudit/blob/master/README.md#format).
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-cosmos-db-for-postgresql.md:18]:
```suggestion
* **Audit Logs**. Audit logging of database activities is available through [pgAudit](https://www.pgaudit.org/) extension. By default, pgAudit log statements are emitted along with your regular log statements by using Postgres's standard logging facility. To know more about the audit log format, refer to the [pgAudit documentation](https://github.com/pgaudit/pgaudit/blob/master/README.md#format).
```

File: docs/integrations/microsoft-azure/azure-cosmos-db-for-postgresql.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+* **Metrics**. These metrics are available for every node of a cluster, and in aggregate across the nodes.For more information on supported metrics, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/cosmos-db/postgresql/concepts-monitoring#list-of-metrics).
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-cosmos-db-for-postgresql.md:19]:
```suggestion
* **Metrics**. These metrics are available for every node of a cluster, and in aggregate across the nodes. For more information on supported metrics, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/cosmos-db/postgresql/concepts-monitoring#list-of-metrics).
```

File: docs/integrations/microsoft-azure/azure-database-for-postgresql.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+description: Learn about the Sumo Logic collection process for the Azure Database for PostgreSQL service.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/azure-database-for-postgresql.png')} alt="Thumbnail icon" width="50"/>
+
+[Azure Database for PostgreSQL](https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/overview) is a fully managed relational database service in the Microsoft cloud based on the PostgreSQL community edition. This integration helps in monitoring resource utilization and identifying slow queries to optimize your workloads and configure your server for the best performance.
+
+The below instructions applies to Azure Database for PostgreSQL - Flexible Server as Azure Database for PostgreSQL - Single Server is on the retirement path.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-database-for-postgresql.md:13]: Can you be more clear here, it is confusing with two hypens.

File: docs/integrations/microsoft-azure/azure-database-for-postgresql.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+## Log and Metric types
+
+For Azure Database for PostgreSQL, you can collect the following logs and metrics:
+
+* **PostgreSQL Logs**. These logs can be used to identify, troubleshoot, and repair configuration errors and suboptimal performance. To know more about the log format, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/concepts-logging#log-format).
+* **Audit Logs**. Audit logging of database activities is available through [pgAudit](https://www.pgaudit.org/) extension.By default, pgAudit log statements are emitted along with your regular log statements by using Postgres's standard logging facility. To know more about the audit log format, refer to the [pgAudit documentation](https://github.com/pgaudit/pgaudit/blob/master/README.md#format).
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-database-for-postgresql.md:20]:
```suggestion
* **Audit Logs**. Audit logging of database activities is available through [pgAudit](https://www.pgaudit.org/) extension. By default, pgAudit log statements are emitted along with your regular log statements by using Postgres's standard logging facility. To know more about the audit log format, refer to the [pgAudit documentation](https://github.com/pgaudit/pgaudit/blob/master/README.md#format).
```

File: docs/integrations/microsoft-azure/azure-machine-learning.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+id: azure-machine-learning
+title: Azure Machine Learning
+description: Learn about the Sumo Logic collection process for the Azure Machine Learning service.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/azure-machine-learning.png')} alt="Thumbnail icon" width="50"/>
+
+[Azure Machine Learning](https://learn.microsoft.com/en-us/azure/machine-learning/overview-what-is-azure-machine-learning) is a cloud service for accelerating and managing the machine learning project lifecycle which includes training, deploying and monitoring models. This integration helps in monitoring the training runs, model deployments, and resource utilization of your nodes present in your workspace.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-machine-learning.md:11]:
```suggestion
[Azure Machine Learning](https://learn.microsoft.com/en-us/azure/machine-learning/overview-what-is-azure-machine-learning) is a cloud service for accelerating and managing the machine learning project lifecycle which includes training, deploying, and monitoring models. This integration helps in monitoring the training runs, model deployments, and resource utilization of your nodes present in your workspace.
```

File: docs/integrations/microsoft-azure/azure-machine-learning.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+## Log and Metric types
+
+For Azure Machine Learning, you can collect the following logs and metrics:
+
+* **Resource logs**. These logs contains information about clusters, nodes, jobs, deployments, models and pipeline events. To know more about the different [log category types](https://learn.microsoft.com/en-us/azure/machine-learning/monitor-azure-machine-learning?view=azureml-api-2#analyzing-logs) and [schemas](https://learn.microsoft.com/en-us/azure/machine-learning/monitor-resource-reference?view=azureml-api-2#schemas) collected for Azure Machine Learning, refer to the documentation.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-machine-learning.md:17]:
```suggestion
* **Resource logs**. These logs contains information about clusters, nodes, jobs, deployments, models, and pipeline events. To know more about the different [log category types](https://learn.microsoft.com/en-us/azure/machine-learning/monitor-azure-machine-learning?view=azureml-api-2#analyzing-logs) and [schemas](https://learn.microsoft.com/en-us/azure/machine-learning/monitor-resource-reference?view=azureml-api-2#schemas) collected for Azure Machine Learning, refer to the documentation.
```

File: docs/integrations/microsoft-azure/azure-relay.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+<img src={useBaseUrl('img/integrations/microsoft-azure/azure-relay.png')} alt="Thumbnail icon" width="50"/>
+
+The [Azure Relay](https://learn.microsoft.com/en-us/azure/azure-relay/relay-what-is-it) service enables you to securely expose services that run in your corporate network to the public cloud. This integration helps in monitoring  everything that happens with operations and actions that are conducted against your Azure Relay namespace by using the API, or through language SDK.
+
+## Log and Metric types
+
+For Azure Relay, you can collect the following logs and metrics:
+
+* **Diagnostic Logs**. To know more about the different resource log category types and schemas collected for Azure Relay, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/azure-relay/diagnostic-logs).
+
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-relay.md:18]:
```suggestion
```

File: docs/integrations/microsoft-azure/azure-service-bus.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+id: azure-service-bus
+title: Azure Service Bus
+description: Learn about the Sumo Logic collection process for the Azure Service Bus service.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/azure-service-bus.png')} alt="Thumbnail icon" width="50"/>
+
+[Azure Service Bus](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview) is a fully managed enterprise message broker with message queues and publish-subscribe topics (in a namespace) used for decoupling applications and services from each other. This integration helps in monitoring incoming/outgoing messages, connections, throttled requests, resource usage of your Service Bus namespace.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-service-bus.md:11]:
```suggestion
[Azure Service Bus](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview) is a fully managed enterprise message broker with message queues and publish-subscribe topics (in a namespace) used for decoupling applications and services from each other. This integration helps in monitoring incoming/outgoing messages, connections, throttled requests, and resource usage of your Service Bus namespace.
```

File: docs/integrations/microsoft-azure/azure-stream-analytics.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+<img src={useBaseUrl('img/integrations/microsoft-azure/azure-stream-analytics.png')} alt="Thumbnail icon" width="50"/>
+
+[Azure Stream Analytics](https://learn.microsoft.com/en-us/azure/stream-analytics/stream-analytics-introduction) is a fully managed stream processing engine that is designed to analyze and process large volumes of streaming data with sub-millisecond latencies. This integration helps in monitoring and troubleshooting your query and job performance.
+
+## Log and Metric types
+
+For Azure Stream Analytics, you can collect the following logs and metrics:
+
+* **Resource logs**. Azure Stream Analytics captures below two categories of resource logs:
+  * Authoring: These log events that are related to job authoring operations, such as job creation, and starting or stopping the job.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-stream-analytics.md:18]:
```suggestion
  * **Authoring**. These log events that are related to job authoring operations, such as job creation, and starting or stopping the job.
```

File: docs/integrations/microsoft-azure/azure-stream-analytics.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+  * Execution: These events are related to job execution such as connectivity and data processing errors.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-stream-analytics.md:19]:
```suggestion
  * **Execution**. These events are related to job execution such as connectivity and data processing errors.
```

File: docs/integrations/microsoft-azure/azure-sql-managed-instance.md [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+
+[Azure SQL Managed Instance](https://learn.microsoft.com/en-us/azure/azure-sql/managed-instance/sql-managed-instance-paas-overview) is a scalable cloud database service that's always running on the latest stable version of the Microsoft SQL Server database engine and a patched OS with 99.99% built-in high availability, offering close to 100% feature compatibility with SQL Server. This integration helps in monitoring resource usage and tracking database events of your managed instances.
+
+## Log and Metric types
+
+For Azure SQL Managed Instance, you can collect the following logs and metrics:
+
+* **Resource Usage Statistics**. To know more about the different [log category types](https://learn.microsoft.com/en-us/azure/azure-sql/database/metrics-diagnostic-telemetry-logging-streaming-export-configure?tabs=azure-portal&view=azuresql#instances-in-azure-sql-managed-instance) and [schemas](https://learn.microsoft.com/en-us/azure/azure-sql/database/metrics-diagnostic-telemetry-logging-streaming-export-configure?tabs=azure-portal&view=azuresql#resource-usage-stats-for-managed-instances) collected for Azure SQL Managed Instances, refer to the Azure documentation.
+
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-sql-managed-instance.md:18]:
```suggestion
```

File: docs/integrations/microsoft-azure/azure-sql-elastic-pool.md [hunk 14]
```diff
@@ -0,0 +1,44 @@
...
+
+When you configure the event hubs source or HTTP source, plan your source category to ease the querying process. A hierarchical approach allows you to make use of wildcards. For example:  `Azure/SQLElasticPool/Metrics`.
+
+### Configure metrics collection
+
+In this section, you will configure a pipeline for shipping metrics from Azure Monitor to an Event Hub, on to an Azure Function, and finally to an HTTP Source on a hosted collector in Sumo Logic.
+
+1. [Configure an HTTP Source](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-1-configure-an-http-source).
+2. [Configure and deploy the ARM Template](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-2-configure-azure-resources-using-arm-template).
+3. [Export metrics to Event Hub](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-3-export-metrics-for-a-particular-resource-to-event-hub). Perform below steps for each Azure SQL Elastic pool that you want to monitor.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-sql-elastic-pool.md:35]:
```suggestion
3. [Export metrics to Event Hub](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-3-export-metrics-for-a-particular-resource-to-event-hub). Perform below steps for each Azure SQL Elastic Pool that you want to monitor.
```

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor fix

File: docs/integrations/microsoft-azure/azure-database-for-postgresql.md (hunk 5 above)
Comment (himanshu219) [reply docs/integrations/microsoft-azure/azure-database-for-postgresql.md:13]: so basically Azure Database for PostgreSQL - Flexible Server and Azure Database for PostgreSQL - Single Server are separate configurations for this database.

Comment (himanshu219) [reply docs/integrations/microsoft-azure/azure-database-for-postgresql.md:13]: changing this to "applies to Azure Database for PostgreSQL with Flexible Server only."

**Review by JV0812 — APPROVED** [review]
LGTM

**Pull request merged**